docs/blocks/reference.md

```diff
@@ -0,0 +1,220 @@
+Method Reference
```

Comment:
So, I think this is amazing, but I also think it shouldn't be in the docs. This should be within Parchment or NeoForge itself since you are literally providing a Javadoc here. If there are methods that are replaced by NeoForge, then it would make sense to have a brief mention here with an example.

Reply:
Removed from this PR, will PR to Parchment once I find the time.